common/pom.xml

```diff
@@ -338,22 +333,10 @@
                         </manifestEntries>
                     </archive>
                 </configuration>
-            </plugin>
-
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
-                <version>2.4</version>
-                <configuration>
-                    <archive>
-                        <manifestEntries>
-                            <Go-Version>${go.version}</Go-Version>
```

Comment:
This is removing the dup from here https://github.com/gocd/gocd/blob/3b6aac252eef6061016e5cecde6aa73ef5da04d0/common/pom.xml#L343-L362, which is why you see an additional `<goal>jar</goal>` in the lines that follow. Ditto with `config/config-api/pom.xml`